format/prettier.config.json

```diff
@@ -0,0 +1,19 @@
+{
+  "arrowParens": "always",
+  "bracketSpacing": true,
+  "embeddedLanguageFormatting": "auto",
+  "htmlWhitespaceSensitivity": "css",
+  "insertPragma": false,
+  "jsxBracketSameLine": false,
+  "jsxSingleQuote": false,
+  "printWidth": 80,
```

Comment:
I don't do a lot of JS coding so I don't care for a particular formatter, but I generally find it too narrow to use 80 chars per line and 2-space indent. I don't have any objections though, just that my screen fits more, and I feel like there's a bit more line-wrapping in odd places happening now than before.

Reply:
How about 100 chars? I've increased it in 86251715cb91ae88f290a91b3d747c443d350024.